android/src/main/kotlin/io/dolby/comms/sdk/flutter/module/ConferenceServiceNativeModule.kt

```diff
@@ -79,14 +80,18 @@ class ConferenceServiceNativeModule(private val scope: CoroutineScope) : NativeM
     private fun create(call: MethodCall, result: Result) = scope.launch(
         onError = result::error,
         onSuccess = {
-            val paramsHolder = HashMap<String, Any?>()
-            call.argument<Map<String, Any>>("params")?.let {
-                paramsHolder.putAll(it)
+            val params = HashMap<String, Any?>()
+            call.argument<Map<String, Any>>("params")?.let { params.putAll(it) }
+
+            val paramsHolder = ParamsHolder()
```

Comment:
Don't we need to set the `params` in the `paramsHolder` ?

Reply:
change to: val paramsHolder = ParamsHolder(maps)